.github/workflows/run-test.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under the MIT license, see file COPYING for the full text
+
+name: Run tests
+
+on:
+- pull_request
+- push
+
+jobs:
+  run-tests:
+    name: Run tests
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.4.0
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: '3.10'
```

Comment:
I'm running debian stable both locally and in production. Therefore, I would prefer to have py3.9 checked in CI.

Reply:
Okay, done